src/compat/compat.h

```diff
@@ -327,6 +298,9 @@ get_cluster_options(const ClusterStmt *stmt)
 	return stmt->options;
 }
 #else
+
+#include <commands/defrem.h>
```

Comment:
Is this chance really necessary?

Reply:
while the code does work without the change it adjusts the include to only pull it in when it is required (when we have to fall back to our own get_cluster_options implementation